front/app/containers/App/index.tsx

```diff
@@ -588,6 +614,8 @@ class App extends PureComponent<Props, State> {
 const Data = adopt<DataProps, InputProps>({
   windowSize: <GetWindowSize />,
   redirectsEnabled: <GetFeatureFlag name="redirects" />,
+  // CL-1101, FranceConnect platforms have full screen login experience
+  fullscreenModalEnabled: <GetFeatureFlag name="franceconnect_login" />,
```

Comment:
Currently, we check the `franceconnect_login` feature flag twice (here and in `SignUpInModal`). Maybe we could pass `fullscreenModalEnabled` into `SignUpInModal` to get the feature flag only once. Also, it will bond this logic in two places together from one place in the code. Because it seems to be just one the same condition by its nature.

1. It makes it easier to understand that these two places where we check the `franceconnect_login` feature are really connected.
2. It makes it easier to change this condition in the future.